styleguides/php/core.md

```diff
@@ -23,14 +19,6 @@ The below documentation links should be followed unless defined otherwise in thi
 
    * [PSR-1](https://www.php-fig.org/psr/psr-1/)
    * [PSR-2](https://www.php-fig.org/psr/psr-2/)
-   
-#### Laravel
```

Comment:
Before removing the Laravel docs and the RESTful API standards these should be moved somewhere else